docs/explanation/registry_key_concepts.md

```diff
@@ -0,0 +1,203 @@
+
+Finally, separating static data from dynamic data allows for accurate representation and easy updates. Static data, such as names and birthdates, are stored in the core registry, while dynamic data, like income changes or program participation, are managed as {term}`event data`. This separation keeps the core registry uncluttered and efficient, facilitating better management and updates.
+
+By adopting these practices, OpenSPP ensures secure, efficient, and scalable data management, effectively supporting diverse program needs while maintaining a robust core infrastructure.
+
+## Deciding Where to Store Data in OpenSPP
+
+To effectively decide where different types of data should be stored in OpenSPP, it is essential to categorize the data into one of the following five categories: static core registry, dynamic core registry, static program, dynamic program, or not needed to be stored. This decision process ensures optimal data management, security, and efficiency. Here's a guide to help make these decisions:
+
+1. **Static Core Registry**
+   - **Characteristics:** Information that is unlikely to change over time or that does not change often and is universally relevant across all programs.
+   - **Examples:** Names, birthdates, national identification numbers, contact details, marital status, address.
+   - **Guidelines:**
+     - Does the data remain constant throughout an individual's lifetime?
+     - Is the data required by all programs for identification or basic profiling?
+
+2. **Dynamic Core Registry**
+   - **Characteristics:** Information that changes periodically and is relevant across multiple programs.
+   - **Examples:** Household income level, poverty score.
+   - **Guidelines:**
+     - Does the data change but is still needed universally by all programs?
+     - Is it important to keep this data updated centrally for all programs to access?
+
+3. **Static Program**
+   - **Characteristics:** Information that is specific to a particular program and does not change frequently.
+   - **Examples:** Enrollment date in a specific program, initial eligibility criteria.
+   - **Guidelines:**
+     - Is the data relevant only to a specific program?
+     - Is it unlikely to change once recorded?
+
+4. **Dynamic Program**
+   - **Characteristics:** Information that is specific to a particular program and changes over time.
+   - **Examples:** Program participation status, progress updates.
+   - **Guidelines:**
+     - Is the data relevant only to a specific program and likely to change over time?
+     - Does the program need to track updates or changes in this data?
+
+5. **Not Needed to be Stored**
+   - **Characteristics:** Information that is either unnecessary to store or can be derived from other data.
+   - **Examples:** Temporary notes, redundant data, derived metrics.
+   - **Guidelines:**
+     - Is the data critical for decision-making or program operation?
```

Comment:
@jeremi I think the questions `Is the data critical for decision-making or program operation?` and `Does storing this data comply with data minimization principles?` should be asked for 1, 2, 3, and 4.

I'd suggest we use the opposite wording for section 5.

Reply:
I will add them :)